array-manipulation-1/src/reverse.js

```diff
@@ -0,0 +1,19 @@
+/* exported reverse */
+// 1. define function reverse with perameter array
+// 2. if argument array is equal to empty array
+// 3. return empty array
+// 4. assign empty array to variable final array
+// 5. loop through array indexes backwards
+// 6. push array[i] to finalArray array
+// 7. return finalArray
+function reverse(array) {
+  if (array === []) {
+    return [];
+  }
```

Comment:
This condition will never be true. Arrays and objects are never equal to each other.